website/content/en/docs/troubleshooting.md

```diff
@@ -284,7 +284,22 @@ Karpenter does not support [in-tree storage plugins](https://kubernetes.io/blog/
 
 #### Pods were scheduled due to a race condition in Kubernetes
 
-Due to [this race condition in Kubernetes](https://github.com/kubernetes/kubernetes/issues/95911), it's possible that the scheduler and the CSINode can race during node registration such that the scheduler assumes that a node can mount more volumes than the node attachments support. There is currently no solve for this problem other than enforcing `toplogySpreadConstraints` and `podAntiAffinity` on your workloads that use PVCs such that you attempt to reduce the number of PVCs that schedule to a given node.
+Due to [this race condition in Kubernetes](https://github.com/kubernetes/kubernetes/issues/95911), it's possible that the scheduler and the CSINode can race during node registration such that the scheduler assumes that a node can mount more volumes than the node attachments support. There is currently no universal solve for this problem other than enforcing `toplogySpreadConstraints` and `podAntiAffinity` on your workloads that use PVCs such that you attempt to reduce the number of PVCs that schedule to a given node.
+
+The following is a non-comprehensive list of CSI drivers which support a startupTaint to eliminate this issue:
```

Comment:
```suggestion
The following is a known list of CSI drivers which support a startupTaint to eliminate this issue:
```

nit: It sounds like we don't know or we didn't do our work to find out the detail if we say something like "non-comprehensive". I'd go for something like "known" and just accept the fact that this list may expand as other see it and think, "wait, I know of another CSI driver that does this" and they add it

Reply:
Probably fair, I said non-comprehensive because I am sure other CSI drivers may have this feature and didn't want to lead users to believe that these are the only two drivers that do. I think known is probably a good middle ground.